includes/definitions/discovery/zxr10.yaml

```diff
@@ -0,0 +1,13 @@
+mib: ZXR10-MIB
+modules:
+    mempools:
+        data:
+            -
+                total: ZXR10-MIB::zxr10SystemMemSize.1
+                percent_used: ZXR10-MIB::zxr10SystemMemUsed.1
```

Comment:
The mib says zxr10SystemMemUsed is in bytes not percent.  so you should use "used:" instead.
You should also remove the precision line below.

Reply:
I thought so too, but:
$ snmpwalk ....... zxr10SystemMemSize
ZXR10-MIB::zxr10SystemMemSize.1 = INTEGER: 268435456
mem-size - 256M in bytes

$ snmpwalk ...... zxr10SystemMemUsed
ZXR10-MIB::zxr10SystemMemUsed.1 = INTEGER: 60
mem-used is in percent not in bytes

zxr10 device cli shows the same:
#show processor 
PhyMem: Physical memory (megabyte)

    Panel    CPU(5s) CPU(1m) CPU(5m)  PhyMem  Buffer  Memory
           
MP(M)   1      47%     48%     48%      256     0%      59.958%

Reply:
I removed the precision line and now memory draws correctly
And now memory oids collects without .1 index.

Reply:
Good job following your own mib ZTE :D  Looks good.